cpp/src/slice2cs/Gen.cpp

```diff
@@ -653,657 +713,98 @@ Slice::CsVisitor::writeDataMemberInitializers(const DataMemberList& dataMembers,
     }
 }
 
-string
-Slice::CsVisitor::toCsIdent(const string& s)
-{
-    string::size_type pos = s.find('#');
-    if (pos == string::npos)
-    {
-        return s;
-    }
-
-    string result = s;
-    if (pos == 0)
-    {
-        return result.erase(0, 1);
-    }
-
-    result[pos] = '.';
-    return result;
-}
-
-string
-Slice::CsVisitor::editMarkup(const string& s)
```

Comment:
I need add the logic for `&lt;` and `&gt;` but would be better to move it to parseDocComment.